qcodes/tests/dataset/test_data_set_cache.py

```diff
@@ -122,19 +128,29 @@ def test_cache_1d(experiment, DAC, DMM, n_points, bg_writing,
                                                     shaped_partial=set_shape)
     _assert_parameter_data_is_identical(dataset1.get_parameter_data(),
                                         dataset1.cache.data())
-    assert dataset1.cache._loaded_from_completed_ds is True
+    if in_memory_cache is False:
+        assert dataset1.cache._loaded_from_completed_ds is True
+    else:
+        # todo enable live cache to set this
+        assert dataset1.cache._loaded_from_completed_ds is False
```

Comment:
do i understand it correctly?
```suggestion
        assert dataset1.cache._loaded_from_completed_ds is False
    assert dataset1.cache.live is in_memory_cache
```